openmls/src/schedule/psk.rs

```diff
@@ -141,16 +154,18 @@ pub enum Psk {
 /// in the key schedule.
 ///
 /// ```c
-/// // draft-ietf-mls-protocol-16
+/// // draft-ietf-mls-protocol-17
```

Comment:
I still don't think that these make sense.
But if you actually want to update and use them, you should use the current version -18.

Reply:
Checked that `PSKType`, `ResumptionPSKUsage`, and `PreSharedKeyID` are still the same in -18. Updated.